pkgs/top-level/ocaml-packages.nix

```diff
@@ -522,6 +522,8 @@ let
 
     ocaml_sqlite3 = callPackage ../development/ocaml-modules/sqlite3 { };
 
+    ocaml_syslog = callPackage ../development/ocaml-modules/syslog { };
```

Comment:
I am unsure about this

Reply:
I think that  `syslog` is enough.

Reply:
Changed. TBH beside the symbolic name of the packages I have no idea what this ocamlPackages attribute set is used for.